xwiki-platform-core/xwiki-platform-image/xwiki-platform-image-lightbox/xwiki-platform-image-lightbox-ui/src/main/resources/XWiki/Lightbox/WebHome.xml

```diff
@@ -0,0 +1,654 @@
+  aria-modal="true"
+  role="dialog"
+&gt;
+  &lt;div class="slides" aria-live="polite"&gt;&lt;/div&gt;
+  &lt;a
+    class="prev" aria-controls="blueimp-gallery"
+    aria-label="$escapetool.xml($services.localization.render('image.lightbox.prevSlide.title'))"
+    title="$escapetool.xml($services.localization.render('image.lightbox.prevSlide.title'))"
+    aria-keyshortcuts="ArrowLeft"
+  &gt;&lt;/a&gt;
+  &lt;a
+    class="next" aria-controls="blueimp-gallery"
+    aria-label="$escapetool.xml($services.localization.render('image.lightbox.nextSlide.title'))"
+    title="$escapetool.xml($services.localization.render('image.lightbox.nextSlide.title'))"
+    aria-keyshortcuts="ArrowRight"
+  &gt;&lt;/a&gt;
+  &lt;div class="controls btn-group"&gt;
+    &lt;a
+      class="btn fullscreen" id="lightboxFullscreen" aria-controls="blueimp-gallery"
+      aria-label="$escapetool.xml($services.localization.render('image.lightbox.fullscreen.title'))"
+      title="$escapetool.xml($services.localization.render('image.lightbox.fullscreen.title'))"
+      aria-keyshortcuts="F11"
+    &gt;$services.icon.renderHTML('arrows')&lt;/a&gt;
+    &lt;a
+      class="btn download" id="lightboxDownload" aria-controls="blueimp-gallery" target="_blank"
+      aria-label="$escapetool.xml($services.localization.render('image.lightbox.download.title'))"
+      title="$escapetool.xml($services.localization.render('image.lightbox.download.title'))"
+    &gt;$services.icon.renderHTML('download')&lt;/a&gt;
+    &lt;a
+      class="btn autoPlay" aria-controls="blueimp-gallery"
+      aria-label="$escapetool.xml($services.localization.render('image.lightbox.autoPlay.title'))"
+      title="$escapetool.xml($services.localization.render('image.lightbox.autoPlay.title'))"
+      aria-keyshortcuts="Space" aria-pressed="false"
+    &gt;$services.icon.renderHTML('play')&lt;/a&gt;
+    &lt;a
+      class="btn escape" aria-controls="blueimp-gallery"
+      aria-label="$escapetool.xml($services.localization.render('image.lightbox.escape.title'))"
+      title="$escapetool.xml($services.localization.render('image.lightbox.escape.title'))"
+      aria-keyshortcuts="Escape"
+    &gt;$services.icon.renderHTML('cross')&lt;/a&gt;
+  &lt;/div&gt;
+  &lt;div class="lightboxDescription"&gt;
+    &lt;div class="caption"
+      aria-label="$escapetool.xml($services.localization.render('image.lightbox.caption.label'))"&gt;&lt;/div&gt;
+    &lt;div class="descriptionMetadata"&gt;
+      &lt;span class="title"
+        aria-label="$escapetool.xml($services.localization.render('image.lightbox.fileName.label'))"&gt;&lt;/span&gt;
+      &lt;div class="descriptionRow"&gt;
+        &lt;span class="publisher"
+          aria-label="$escapetool.xml($services.localization.render('image.lightbox.publisher.label'))"&gt;&lt;/span&gt;
+        &lt;span class="date" data-dateFormat="$dateFormat"
+          aria-label="$escapetool.xml($services.localization.render('image.lightbox.date.label'))"&gt;&lt;/span&gt;
```

Comment:
Where is that translation key defined?

Reply:
Forgot to add the change, thanks! Adding it